src/server/plugins/register-form/steps/date-of-birth.js

```diff
@@ -0,0 +1,33 @@
+import Joi from 'joi';
+import {postHandlerFactory, getHandlerFactory} from './common';
+
+const fields = [
+  {id: 'day', label: 'Day'},
+  {id: 'month', label: 'Month'},
+  {id: 'year', label: 'Year'}
+];
+
+const schema = Joi.object().keys({
+  'day': Joi.number().integer().min(1).max(31).required(),
+  'month': Joi.number().integer().min(1).max(12).required(),
+  'year': Joi.number().integer().min(1885).max(2025).required(),
+  'submit': Joi.any().optional().strip()
+});
+
+const title = 'What is your date of birth?';
+const key = 'dateOfBirth';
+
+const handlers = {
+  GET: getHandlerFactory(key, fields, title, schema),
+  POST: postHandlerFactory(key, fields, title, schema, 'dateOfBirth')
```

Comment:
want to have this `nextStep` parameter provided from the order of the `steps` definition